.github/CODEOWNERS

```diff
@@ -47,11 +47,11 @@
 /tests/tests_fabric                         @awaelchli @carmocca @justusschock
 
 # Lightning App
-/src/lightning/app                          @tchaton @lantiga @awaelchli @ethanwharris
-/src/lightning_app                          @tchaton @lantiga @awaelchli @ethanwharris
-/tests/tests_app                            @tchaton @lantiga @awaelchli @ethanwharris
-/tests/integrations_app                     @tchaton @lantiga @awaelchli @ethanwharris
-/examples/app_*                             @tchaton @lantiga @awaelchli @ethanwharris
+/src/lightning/app                          @awaelchli @ethanwharris @lantiga @tchaton
```

Comment:
alphabetic order

Reply:
I don't think I should be first. Thomas knows more than me in that code base. The ordering should probably reflect that

Reply:
alphabetic